tests/cue/env.py

```diff
@@ -6,36 +6,19 @@
 class VSCEnvTest(rfm.RunOnlyRegressionTest):
     descr = "test environment variable "
     envar = parameter(['VSC_HOME', 'VSC_DATA', 'VSC_SCRATCH'])
-    valid_systems = required
-    valid_prog_environs = required
+    valid_systems = ["*"]
+    valid_prog_environs = ["*"]
```

Comment:
Similar to the remark for the line above, I think it suffices to run this test only for the `["builtin"]` programming environment.